### PR TITLE
Feature/chris cost estimator

### PR DESCRIPTION
**Pull request by ChrisMolanus**
Working streamlit app

### CODE REVIEW

**Review by HSCunha — APPROVED** [review]
Reviewed and approved for merging these changes.

File: ftth_app.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+north = north_field.text_input('North','50.78694')
+south = south_field.text_input('South','50.77902')
+east = east_field.text_input('East', '4.48386')
+west = west_field.text_input('West', '4.49521')
+
+detailed_cost, fig = get_planning()
+
+# Map with optimal fiber route
+st.subheader(f'Optimal fiber network route for [N:{north}, S:{south}, E:{east}, W:{west}] \n')
+st.pyplot(fig)
```
Comment (tiaan524) [inline ftth_app.py:51]: Insert legend on app. (street cabinets, fiber network, different types of fibers)

**Review by tiaan524 — APPROVED** [review]

File: testnetwork.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+from trenches2 import get_trench_network
```
Comment (tivoo) [inline testnetwork.py]: this file can be deleted

**Comment by tivoo** [comment]
streamlit module should part of requirements.txt

**Review by tivoo — COMMENTED** [review]
Added some comments and conflict in fibers.py needs to be double checked. Otherwise looking fine.

**Review by HSCunha — APPROVED** [review]
Done!

**Pull request merged**